pages/blog/index.vue

```diff
@@ -8,45 +8,67 @@
           </NuxtLink>
         </Center>
       </UContainer>
-      <UContainer class="py-10" :ui="{'constrained': 'max-w-2xl'}">
-        <h1>Neues aus der digitalen Welt</h1>
-        <p class="lead">Hier möchten wir gerne unser Wissen, über die digitale Zukunft, Technologien, Design und das Leben in einer digitalen Agentur, mit euch teilen, bleibt gespannt, wir sind es auch.</p>
+      <UContainer class="pt-16" :ui="{'constrained': 'max-w-7xl'}">
+        <Headline type="h2" class="pb-8 leading-[57px] text-[48px] lowercase">
+          <b class="text-jm-primary-brown uppercase">Neues</b> aus der
+          <b class="text-jm-primary-brown uppercase"> digitalen Welt </b>
+        </Headline>
+        <Paragraph class="text-base mb-8">Hier möchten wir gerne unser Wissen, über die digitale Zukunft, Technologien, Design <br> und das Leben in einer digitalen Agentur, mit euch teilen, bleibt gespannt, wir sind es auch.</Paragraph>
+        <UContainer :ui="{'constrained': 'max-w-4xl space-x-4 ml-0 lg:px-0'}">
+          <Button
+              v-for="(category, index) in categories"
+              :key="index"
+              @click="selectedCategory = category"
+          class="text-jm-primary-brown border-jm-primary-brown"
+          >{{category}}</Button>
+        </UContainer>
 
-        <Button>Alle anzeigen</Button>
-        <Button>Design</Button>
-        <Button>Entwicklung</Button>
-        <Button>Neuigkeiten</Button>
-
-        <UBlogList>
-          <UBlogPost v-for="(article, index) in articles" :key="index" v-bind="article" />
+        <UBlogList orientation="horizontal" class="mt-12">
+          <UBlogPost
+              v-for="(article, index) in filteredArticles" :key="index"
+                     class="bg-jm-secondary-grey-lighter ">
+            <NuxtLink :to="article._path">
+            <NuxtImg :src="article.image" format="webp"/>
+            <section class="px-7 pb-5">
+            <Paragraph class="mt-4 mb-2 text-[14px] font-light">{{article.date}} von <b class="text-jm-primary-green uppercase"> {{article.author}} </b></Paragraph>
+            <Headline class="font-extrabold text-lg" type="h5" v-html="article.title"/>
+            <Paragraph class="text-sm mb-4 font-light">{{ truncateText(article.description, 250) }}  </Paragraph>
+            <UBadge
+                v-for="(category, index) in article.categories.slice(1)"
+                :key="index"
+                color="white"
+                class="mr-2 py-0.5 text-[12px] text-jm-secondary-white bg-jm-primary-brown font-extrabold uppercase"
```

Comment:
Please do not use [12px] as discussed ready-made values that come close to that from Tailwind.